docs/README.md

```diff
@@ -60,8 +60,28 @@ Detailed information about MCM SDK can be found in [sdk](sdk) directory.
 ### Basic Installation
 
 0. **Install Dependencies**
-    - MTL: Follow the [MTL setup guide](https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/doc/build.md) for installation.
-    - gRPC: Refer to the [gRPC documentation](https://grpc.io/docs/languages/cpp/quickstart/) for installation instructions.
+
```

Comment:
Zero numbering looks weird
![image](https://github.com/user-attachments/assets/f6e9cf71-f8dd-4ca6-8d02-8cea07b3526c)

Reply:
1. Install dependencies
    - MTL
    - gRPC
    - libbsd
        - Ubuntu/Debian
        - Centos
     - libfabric

2. Clone the repo

Reply:
done